mne/viz/brain/surface.py

```diff
@@ -0,0 +1,191 @@
+        self.curv = None
+        self.faces = None
+        self.grey_curv = None
+        self.nn = None
+
+        subjects_dir = _get_subjects_dir(subjects_dir)
+        self.data_path = path.join(subjects_dir, subject_id)
+
+    def load_geometry(self):
+        """Load geometry of the surface.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        None
+        """
+        from nibabel import freesurfer
+        from surfer.utils import _compute_normals
+
+        surf_path = path.join(self.data_path, 'surf',
+                              '%s.%s' % (self.hemi, self.surf))
+        coords, faces = freesurfer.read_geometry(surf_path)
+        if self.units == 'm':
+            coords /= 1000.
+        if self.offset is not None:
+            if self.hemi == 'lh':
+                coords[:, 0] -= (np.max(coords[:, 0]) + self.offset)
+            else:
+                coords[:, 0] -= (np.min(coords[:, 0]) + self.offset)
+        nn = _compute_normals(coords, faces)
+
+        if self.coords is None:
+            self.coords = coords
+            self.faces = faces
+            self.nn = nn
+        else:
+            self.coords[:] = coords
+            self.faces[:] = faces
+            self.nn[:] = nn
+
+    def __len__(self):
+        """Return number of vertices."""
+        return len(self.coords)
+
+    @property
+    def x(self):
+        return self.coords[:, 0]
+
+    @property
+    def y(self):
+        return self.coords[:, 1]
+
+    @property
+    def z(self):
+        return self.coords[:, 2]
+
+    def load_curvature(self):
+        """Load in curvature values from the ?h.curv file."""
+        from nibabel import freesurfer
+        curv_path = path.join(self.data_path, 'surf', '%s.curv' % self.hemi)
+        self.curv = freesurfer.read_morph_data(curv_path)
+        self.bin_curv = np.array(self.curv > 0, np.int)
+        # morphometry (curvature) normalization in order to get gray cortex
+        # TODO: delete self.grey_curv after cortex parameter
+        # will be fully supported
+        color = (self.curv > 0).astype(float)
+        color = 0.5 - (color - 0.5) / 3
+        color = color[:, np.newaxis] * [1, 1, 1]
+        self.grey_curv = color
+
+    def load_label(self, name):
+        """Load in a Freesurfer .label file.
+
+        Label files are just text files indicating the vertices included
+        in the label. Each Surface instance has a dictionary of labels, keyed
+        by the name (which is taken from the file name if not given as an
+        argument.
+
+        """
+        from nibabel import freesurfer
+        label = freesurfer.read_label(path.join(self.data_path,
+                                                'label',
+                                                '%s.%s.label' %
+                                                (self.hemi, name)))
+        label_array = np.zeros(len(self.x), np.int)
+        label_array[label] = 1
+        try:
```

Comment:
remove the try.
```py
if name in self.labels:
   self.labels[name] = label_array
else:
   self.labels[name] = label_array
```
or 
```py
getattr(a, 'property', 'default value')
```
or 
```py
if hasattr(self, 'labels'):
    self.labels[name] = foo
else
    self.labels = {name: label_array}
```

Reply:
if not hasattr(bbb):
   labels = { bla}|
elif isinstance(self.labels, dict):
   % dict
else: 
   raiseErro

self.labels = dict()